core/integration/storagetest/mutation_logs.go

```diff
@@ -60,7 +62,7 @@ func (mutationLogsTests) TestReadLog(ctx context.Context, t *testing.T, newForTe
 	logID := int64(5) // Any log ID.
 	m, done := newForTest(ctx, t, directoryID, logID)
 	defer done(ctx)
-	// Write ten batches, three entries each.
+	// Write ten batches.
```

Comment:
You deleted this in the place where we create 3 entries, and left it in the place we create 1.

Reply:
Fixed.